tests/connect/test_alias.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from pyspark.sql.functions import col
+
+
+def test_alias(spark_session):
+    # Create DataFrame from range(10)
+    df = spark_session.range(10)
+
+    # Simply rename the 'id' column to 'my_number'
+    df_renamed = df.select(col("id").alias("my_number"))
+
+    # Verify the alias was set correctly
+    assert df_renamed.schema != df.schema, "Schema should be changed after alias"
+
+    # Verify the data is unchanged but column name is different
+    df_pandas = df.toPandas()
+    df_renamed_pandas = df_renamed.toPandas()
+    assert df_pandas["id"].equals(df_renamed_pandas["my_number"]), "Data should be unchanged after alias"
```

Comment:
i'd like to avoid using pandas in our testing. could we just collect to a dict and test that instead? that`s what we're doing with our daft dataframes in a lot of tests.